task.go

```diff
@@ -310,6 +310,7 @@ func (e *Executor) RunTask(ctx context.Context, call taskfile.Call) error {
 	defer release()
 
 	return e.startExecution(ctx, t, func(ctx context.Context) error {
+		e.Logger.VerboseErrf(logger.Magenta, `started task: '%s'`, call.Task)
```

Comment:
All logs should start with `task:`. What about?

```suggestion
		e.Logger.VerboseErrf(logger.Magenta, `task: "%s" started`, call.Task)
```